### PR TITLE
Account for variable length timestamps when stripping logs

### PR DESCRIPTION
**Pull request by dubee**
See issue for details.

Closes https://github.com/apache/incubator-openwhisk-cli/issues/291

### CODE REVIEW

File: commands/util.go [hunk 1]
```diff
@@ -283,7 +283,7 @@ func printFullActivationList(activations []whisk.Activation) {
 func printActivationLogs(logs []string) {
 	for _, log := range logs {
 		if Flags.activation.strip {
-			fmt.Printf("%s\n", log[39:])
+			fmt.Printf("%s\n", log[strings.Index(log, ": ")+2:])
```
Comment (mdeuser) [inline commands/util.go]: or being completely timestamp format agnostic.. ready to ignore future timestamps with embedded ": "
```
log[strings.Index(log, "stdout: ")+strings.Index(log, "stderr: ")+9:]
```

Comment (dubee) [reply commands/util.go]: @mdeuser, would your example break if the user logs `stderr: ` and the pipe being used is `stdout: `, or vice versa?

Ex:
```
"2018-05-02T19:33:32.829992819Z stdout: this is stdout stderr: "
```

Comment (mdeuser) [reply commands/util.go]: good catch.. yeah.. my code example was too compact
```
skip_index := strings.Index(log, "stdout: ")
if skip_index == -1 {
  skip_index = strings.Index(log, "stdout: ")
}
fmt.Printf("%s\n", log[skip_index+8:])
```

Comment (dubee) [reply commands/util.go]: @mdeuser, I think that would result in the same type of problem.

https://play.golang.org/p/cTGf8yO4Krw

Comment (mdeuser) [reply commands/util.go]: https://play.golang.org/p/8zcL24npnPU

Comment (dubee) [reply commands/util.go]: https://play.golang.org/p/ffJ2Vo41S-6

File: commands/util.go [hunk 2]
```diff
@@ -280,14 +280,22 @@ func printFullActivationList(activations []whisk.Activation) {
...
-func printActivationLogs(logs []string) {
+func printStrippedActivationLogs(logs []string) {
 	for _, log := range logs {
-		if Flags.activation.strip {
-			fmt.Printf("%s\n", log[39:])
+		regex := regexp.MustCompile("[a-zA-Z0-9\\s]*(stdout|stderr):\\s(.*)")
+		match := regex.FindStringSubmatch(log)
+
+		if len(match) > 2 && len(match[2]) > 0 {
+			fmt.Printf("%s\n", match[2])
```
Comment (mdeuser) [inline commands/util.go]: what do you think about using a utility function for the individual log strip logic
```
func stripTimestamp(log string) strippedLog string {
    regex := regexp.MustCompile("[a-zA-Z0-9\\s]*(stdout|stderr):\\s(.*)")
    match := regex.FindStringSubmatch(log)
    if len(match) > 2 && len(match[2]) > 0 {
        strippedLog = match[2]
    } else {
        strippedLog = log
    }
    return strippedLog
}
```
then the print function would become
```
func printStrippedActivationLogs(logs []string) {
	for _, log := range logs {
		fmt.Printf("%s\n", stripTimestamp(log))
 	}
 }
```
the unit test would just validate the utility function.

**Pull request merged**